config/config.go

```diff
@@ -25,21 +23,14 @@ func LoadConfiguration() *viper.Viper {
 	config.Token = v.GetString("CONSUL_TOKEN")
 
 	c, err := api.NewClient(config)
-	PanicOnError(err, "")
+	utils.PanicOnError(err, "")
 
 	pair, _, err := c.KV().Get(v.GetString("CONSUL_PATH"), nil)
-	PanicOnError(err, "Invalid consul config.")
+	utils.PanicOnError(err, "Invalid consul config.")
```

Comment:
Fair enough, I think is should be something app-specific in that case. my argument why would be because what if you wanted to not only panic on that error but also log it? the logging mechanism could change on an app by app basis. Not holding this back but just something to think about when adding code to this lib.